### DISCUSSION TITLE
Double arrow is tilted to the left when in a square

### DISCUSSION THREAD

**Discussion by FrozenRosesLP** [varkor/quiver#96] [Q&A] [answered]
Hello,
Is there a way to make the double arrow vertical and not tilted to the left in this picture?

[Link](https://q.uiver.app/?q=WzAsNCxbMCwwLCJcXGJ1bGxldCJdLFswLDIsIlxcYnVsbGV0Il0sWzIsMCwiXFxidWxsZXQiXSxbMiwyLCJcXHRleHR7dmVyeSBsb29vb25nIG5hbWV9Il0sWzAsMV0sWzAsMl0sWzIsM10sWzEsM10sWzUsNywiIiwyLHsic2hvcnRlbiI6eyJzb3VyY2UiOjIwLCJ0YXJnZXQiOjIwfX1dXQ==).

![image](https://user-images.githubusercontent.com/33604070/120068592-b67ea780-c081-11eb-96a8-ef66f3097d45.png)

**Comment by varkor** [comment]
At the moment, there's not a convenient way to do this in **quiver**, but you can modify the output TikZ code:
```latex
\[\begin{tikzcd}
	\bullet && \bullet \\
	\\
	\bullet && {\text{very loooong name}}
	\arrow[from=1-1, to=3-1]
	\arrow[""{name=0, anchor=center, inner sep=0}, from=1-1, to=1-3]
	\arrow[from=1-3, to=3-3]
	\arrow[""{name=1, anchor=center, inner sep=0, pos=0.92}, from=3-1, to=3-3]
	\arrow[shorten <=9pt, shorten >=9pt, Rightarrow, from=0, to=1]
\end{tikzcd}\]
```
Notice the `pos=0.92` in the (empty) arrow label. It would be good to make this possible from inside **quiver** too; it would probably be worth opening a new issue.

**Comment by FruitfulApproach** [comment]
Does widening the diagram not help?

**Comment by varkor** [comment, marked as answer]
By the way, this issue is now addressed by using the "Align source/target" checkboxes:
<img width="316" alt="image" src="https://github.com/user-attachments/assets/9733f7a3-996c-493f-ba0b-0c29d0c3bd2a" />

For instance, [here is your modified diagram](https://q.uiver.app/#q=WzAsNCxbMCwwLCJcXGJ1bGxldCJdLFswLDIsIlxcYnVsbGV0Il0sWzIsMCwiXFxidWxsZXQiXSxbMiwyLCJcXHRleHR7dmVyeSBsb29vb25nIG5hbWV9Il0sWzAsMV0sWzAsMl0sWzIsM10sWzEsM10sWzUsNywiIiwyLHsic2hvcnRlbiI6eyJzb3VyY2UiOjIwLCJ0YXJnZXQiOjIwfSwiZWRnZV9hbGlnbm1lbnQiOnsidGFyZ2V0IjpmYWxzZX19XV0=).
